### DISCUSSION TITLE
Can't catch Curl error thrown for http DELETE to /session/****

### DISCUSSION THREAD

**Discussion by Jestergnet** [php-webdriver/php-webdriver#1035] [Q&A]
Hello, I can't catch Curl error thrown for http DELETE to /session/****. Sometimes it happens that Selenium restarts and connection refused. In this case, the error is not intercepted in the code and the stream crashes.

What do you advise?

Sample code used:
```
use Facebook\WebDriver\Exception\WebDriverCurlException;

try {

} catch (WebDriverCurlException $e) {

}
```

**Comment by alekciy** [comment]
In my projects I use getAllSessions() https://github.com/php-webdriver/php-webdriver/blob/main/lib/Remote/RemoteWebDriver.php#L569 and if session ID don't exists I start new session.
